gateway/1.4/modules/ROOT/nav.adoc

```diff
@@ -4,9 +4,10 @@
 * xref:flex-review-prerequisites.adoc[Reviewing Prerequisites]
 * xref:flex-gateway-set-up.adoc[]
 ** xref:flex-install.adoc[]
-*** xref:stub.adoc[Install as a Linux Service]
-*** xref:stub.adoc[Install in a Docker Container]
-*** xref:stub.adoc[Install as a Kubernetes Ingress Controller]
+// Are we going to break up the install sections?
```

Comment:
I don't think comments work in the nav.

Reply:
Hey, they seem to work okay. I've built locally and all is well. I'd prefer to leave them so that I don't forget to address these changes.